lib/types/json.js

```diff
@@ -18,6 +18,7 @@ var createError = require('http-errors')
 var debug = require('debug')('body-parser:json')
 var read = require('../read')
 var typeis = require('type-is')
+var JSONBig = require('json-bigint')
```

Comment:
By default, JSONBig will only parse to BigNumber if the number overflows the max safe value. It might be better if we default to always parsing to BigNumber, or maybe make it configurable when setting up the middleware.